src/site/stages/build/drupal/assets.js

```diff
@@ -62,7 +68,9 @@ function updateAttr(attr, doc, client) {
     // *.ci.cms.va.gov ENVs don't have AWS URLs.
     const newAssetPath = convertAssetPath(srcAttr);
     const awsURI = usingAWS
-      ? Object.entries(PUBLIC_URLS).find(entry => entry[1] === siteURI)[0]
+      ? Object.entries(PUBLIC_URLS_NO_SCHEME).find(entry =>
+          siteURI.match(entry[1]),
+        )[0]
```

Comment:
Can we check that this is an array before attempting to grab the first element? If an editor types in `http://prodd.cms.va.gov/sites` this won't find an associated property in `PUBLIC_URLS_NO_SCHEME` and will fail just like it does now when we've got `https:`.

Bonus points for making a helpful message output to the console and continuing the build without replacing the URL. :wink:

Reply:
I debated doing this. I think given this feedback, I'll probably pull out that logic and put it into its own method.